### ISSUE TITLE
ComobBox example should not use movies as data

### ISSUE THREAD

**Issue by lbwexler** [exhi/toolbox#77]
No need for us to slavishly imitate Blueprint's example here

**Comment by shravnk** [comment]
See [#80](https://github.com/exhi/toolbox/pull/80). 

I'm wondering if it makes sense to allow custom filtering logic for a ComboBox. Right now ComboBox can only pass a simple lodash startsWith filter to the underlying blueprint component

**Comment by amcclain** [comment]
Will take a look.  

(While we are in this ticket - and as a vegetarian ExHI team member - I am more thrown off by our "select your meat" radio box example...  🐮🐔🐟😄)

**Comment by lbwexler** [comment]
I heard you were a pescatarian -- need to just enable fish, and disable
beef.

**Comment by amcclain** [comment]
Yes that would do it - thanks for considering my personal dietary choices when it comes to form input examples. Good to know this is such an enlightened toolkit we have here.  (Is there a kale emoji yet?)
